rohmu/object_storage/azure.py

```diff
@@ -76,9 +77,10 @@ def __init__(
         azure_cloud=None,
         proxy_info=None,
         notifier: Optional[Notifier] = None,
+        statsd_info: Optional[StatsdConfig] = None,
```

Comment:
q: This field name looks a bit controversial to me. Why is it "info" and not "config"?

Reply:
we have proxy_info, so went with same things